src/content/docs/new-relic-solutions/get-started/intro-new-relic.mdx

```diff
@@ -202,8 +178,13 @@ Capture, organize, and make sense of your data in New Relic, no matter where it
   />
 
 </TechTileGrid>
+3. See [our quick launch guide](/docs/new-relic-solutions/get-started/quick-launch-guide) for a recommended path on how to set up New Relic. If you'd prefer to add monitoring for some specific entities and services, use the [Add your data](https://one.newrelic.com/launcher/nr1-core.settings?pane=eyJuZXJkbGV0SWQiOiJ0dWNzb24ucGxnLWluc3RydW1lbnQtZXZlcnl0aGluZyJ9) UI page. 
 
-If you don't see your technologies or tasks listed here, see a larger list at [New Relic Instant Observability](https://newrelic.com/instant-observability/). There you will find integrations bundled into quickstarts, providing you instant access to pre-built dashboards and alerts specific to your technology.
+If you don't see your technologies or tasks listed here, see the larger list at [New Relic Instant Observability](https://newrelic.com/instant-observability/). You'll find integrations bundled into quickstarts, providing you instant access to pre-built dashboards and alerts specific to your technology.
+
+Once you have data coming into New Relic, learn more about the [New Relic UI](/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one) or [set up alerts](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/introduction-alerts/).
+
+If you're curious about pricing, we have a [blog post set up to help you with cost estimation](https://newrelic.com/blog/nerdlog/estimate-data-cost). 
```

Comment:
Maybe combine this into one paragraph? I'm not sure if it'll look better but curious how it would look